integration/credential_suspension_integration_test.go

```diff
@@ -0,0 +1,233 @@
+package integration
```

Comment:
could this be combined into a `credential_status_integration_test`?

Reply:
it could be, but I opted for a fresh flow, fresh creds, making sure suspension works, didnt want like a revokable cred flow to 'init' something so a suspension would work